src/services/playlistOrderingService.js

```diff
@@ -27,7 +27,7 @@ function defineMidCycleUserOrder(playlistTracks, currentTrackInfo) {
   return pivotUserOrder(userOrder, currentTrackUserPositionInCycle);
 }
 
-function reorderPlaylist(playlistTracks, currentTrack) {
+function definePlaylistTracksOrder(playlistTracks, currentTrack) {
```

Comment:
Does this name make sense? Are we defining the order here? It seems like we are triggering a reorder.

Reply:
> It seems like we are triggering a reorder.

Depends on the meaning of triggering. This function is tanking an array of playlist tracks and a current track as parameters and returns an array with the playlist tracks ordered by an algorithm. It neither call the Spotify API nor schedules anything to run

> Are we defining the order here?

Yes, see above answer